UnityProject/Assets/Scripts/Tilemaps/Behaviours/TilemapDamage.cs

```diff
@@ -595,7 +595,7 @@ private void SpawnGlassShards(Vector3 pos)
 	{
 		//Spawn 2-4 glass shards
 		Spawn.ServerPrefab("GlassShard", pos, count: Random.Range(2, 4),
-			scatterRadius: Spawn.DefaultScatterRadius);
+			scatterRadius: Random.Range(1, 2));
```

Comment:
this will always be 1, read this carefully: https://docs.unity3d.com/2019.2/Documentation/ScriptReference/Random.Range.html
also there will be 2-3 glass shards for the same reason